src/function/arithmetic/rand_function.cpp

```diff
@@ -0,0 +1,27 @@
+#include "common/random_engine.h"
+#include "function/arithmetic/vector_arithmetic_functions.h"
+#include "function/scalar_function.h"
+#include "main/client_context.h"
+
+namespace kuzu {
+namespace function {
+
+using namespace kuzu::common;
+
+struct Rand {
```

Comment:
Would it be a good idea that `rand()` shares the same random engine with `gen_random_uuid()`? and change the seed for one will reset the other as well.

Reply:
As we discussed, the kuzu should share one random seed